extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -44,7 +45,7 @@
   z-index: 1;
```

Comment:
Can you add a `!important` here that we missed? Also on the top, `--thumb-size` is missing it